doc/design.md

```diff
@@ -195,6 +195,19 @@ Covers:
 
 Needs: impl, itest, utest
 
+
+## Create Target Directory if it does not exist
+
+`dsn~create-target-directory~1`
+
+ECM error code report writer should create project `target` directory if it does not exist (see `ProjectReportWriter`).
```

Comment:
```suggestion
ECM error code report writer creates the project `target` directory if it does not exist (see `ProjectReportWriter`).
```

Reply:
I love present tense for requirements - as it anticipates the implementation as if it was already done.